web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -24,7 +35,17 @@ function setupMockAnalytics(): typeof analytics {
 describe("<SidebarCardsList />", () => {
   beforeEach(() => {
     jest.resetAllMocks();
-    useMockBusiness({});
+    const business = generateBusiness({
+      profileData: generateProfileData({
+        industryId: "generic",
+        businessPersona: "STARTING",
+        sectorId: undefined,
```

Comment:
[pebble] Why are these values necessary as the global mocked business data? 

It seems likely unnecessary globally, because all the other tests were passing even before this change.

If it's only necessary for the new test, it should be scoped just for that test. Even so, I have questions about why some of these specific values (like sector?) are impacting the test outcome

Reply:
taking out